scanners/wpscan/values.yaml

```
@@ -31,7 +31,7 @@ parser:
 scanner:
   image:
     # scanner.image.repository -- Container Image to run the scan
-    repository: wpscanteam/wpscan
+    repository: docker.io/securecodebox/scanner-wpsan
```

```suggestion
    repository: docker.io/securecodebox/scanner-wpscan
```
There seems to be a typo 🤔